bumble/pandora/host.py

```diff
@@ -735,6 +738,18 @@ def uuid16_from_str(uuid: str) -> bytes:
                 )
             )
 
+        flag_map = {
+            NOT_DISCOVERABLE: AdvertisingData.BR_EDR_NOT_SUPPORTED_FLAG,
```

Comment:
This seems wrong, the Flags data type is supposed to be a bit set, so when the `le_discoverability_mode` is NOT_DISCOVERABLE, I would expect to have a flag set to `0`

Reply:
Just updated it